client/src/components/about/Aboutbanner.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent } from 'react';
+import { withTheme } from '@material-ui/core/styles';
+import styled from 'styled-components';
+import Box from '@material-ui/core/Box';
+import Container from '@material-ui/core/Container';
+
+import AboutBanner from '../../assets/about.png';
+
+import { VerticalDivider } from '../divider/VerticalDivider';
+import { Typography } from '@material-ui/core';
+
+interface HomeProps {
+  className?: string;
+}
+
+/**
+ * Banner for About page.
+ */
+const UnstyledAboutBanner: FunctionComponent<HomeProps> = ({ className }) => {
+  return (
+    <Box className={className}>
+      <Box className="about" display="flex" alignItems="center">
+        <Container>
+          <Box
+            className="about-text"
+            display="flex"
+            justifyContent="center"
+            alignItems="flex-start"
+            color="white"
+            flexDirection="column"
+          >
+            About KCSA
+          </Box>
+          {/* <Box className="about-text" display="flex" justifyContent="center" alignItems="center"> */}
+
+          {/* </Box> */}
+        </Container>
+      </Box>
+    </Box>
+  );
+};
+
+export const Aboutbanner = withTheme(styled(UnstyledAboutBanner)`
+  .about {
```

Comment:
classname `about` is to broad. 
`about-banner-container` is better

Reply:
I reflected the class name with about-banner-container